src/ui/src/components/Documentation/components/TreeView.tsx

```
@@ -0,0 +1,191 @@
+            setFocusedNodeId(allNodes[currentIndex + 1].id);
+          }
+          break;
+        case 'ArrowUp':
+          e.preventDefault();
+          if (currentIndex > 0) {
+            setFocusedNodeId(allNodes[currentIndex - 1].id);
+          }
+          break;
+        case 'ArrowRight':
+          e.preventDefault();
+          const currentNode = allNodes[currentIndex];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)